apps/test/unit/componentLibrary/DropdownMenuTest.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
```

Comment:
Rename this file as `SimpleDropdownTest`?

Reply:
yep, doing it now locally

Reply:
fixed